packages/front-end/services/utils.tsx

```diff
@@ -18,7 +18,7 @@ export function phaseSummary(
   return (
     <>
       <span className="phase">
-        {phase.phase === "main" ? "" : phase.phase + ", "}
+        {phase?.phase === "main" || !phase?.phase ? "" : phase.phase + ", "}
```

Comment:
If `phase` is undefined, this whole function should just return `null` immediately.